test/unit/rules/lint-no-curly-component-invocation-test.js

```diff
@@ -29,9 +29,18 @@ generateRuleTests({
      {{/each}}`,
     `{{#-in-element}}Hello{{/-in-element}}`,
     `{{#in-element}}Hello{{/in-element}}`,
+    '<MyComponent @arg={{my-helper this.foobar}} />',
+    '<MyComponent @arg="{{my-helper this.foobar}}" />',
+    '<MyComponent {{my-modifier this.foobar}} />',
```

Comment:
Should the single quotes be ticks instead? Or perhaps the ticks should be single quotes (looks like there are both). 🤷‍♂

Reply:
yeah the consistency here is a bit wonky (not introduced by @lifeart in this PR though), we should do a cleanup pass and see about enforcing consistent quoting via eslint...

Reply:
Fixed: https://github.com/ember-template-lint/ember-template-lint/pull/891